qa/tasks/cephfs/test_volumes.py

```diff
@@ -13,6 +14,14 @@ class TestVolumes(CephFSTestCase):
     TEST_SUBVOLUME_PREFIX="subvolume"
     TEST_GROUP_PREFIX="group"
     TEST_SNAPSHOT_PREFIX="snapshot"
+    TEST_FILE_NAME_PREFIX="subvolume_file"
+
+    # for filling subvolume with data
+    CLIENTS_REQUIRED = 1
+
+    # io defaults
+    DEFAULT_FILE_SIZE = 1 # MB
+    DEFAULT_NUMBER_OF_FILES = 1024
```

Comment:
Do we need so many files? Maybe 100 files is good enough? The test_async_subvolume_rm() failed locally (vstart_runner on vstart_cluster) having timed out.
```
2019-06-21 22:25:39,396.396 INFO:__main__:Running ['getfattr', '--only-values', '-n', 'ceph.dir.entries', '/tmp/tmpg2wMfb/mnt.0/./volumes/_deleting']
2019-06-21 22:25:40,421.421 INFO:__main__:test_async_subvolume_rm (tasks.cephfs.test_volumes.TestVolumes) ... ERROR
2019-06-21 22:25:40,422.422 ERROR:__main__:Traceback (most recent call last):
  File "/home/rraja/git/ceph/qa/tasks/cephfs/test_volumes.py", line 382, in test_async_subvolume_rm
    self._wait_for_trash_empty()
  File "/home/rraja/git/ceph/qa/tasks/cephfs/test_volumes.py", line 69, in _wait_for_trash_empty
    self.mount_a.wait_for_dir_empty(trashdir)
  File "/home/rraja/git/ceph/qa/tasks/cephfs/mount.py", line 255, in wait_for_dir_empty
    i, dirname, self.client_id))
RuntimeError: Timed out after 30s waiting for ./volumes/_deleting to become empty from 0
```